sormas-ui/src/main/java/de/symeda/sormas/ui/ActivityAsCase/ActivityAsCaseField.java

```diff
@@ -230,16 +210,12 @@ public void setPropertyDataSource(Property newDataSource) {
 		}
 	}
 
-	public void setEpiDataParentClass(Class<? extends EntityDto> epiDataParentClass) {
-		this.epiDataParentClass = epiDataParentClass;
-	}
-
 	public void setPseudonymized(boolean isPseudonymized) {
 		this.isPseudonymized = isPseudonymized;
 	}
 
 	private void updateAddButtonVisibility(int activityAsCaseCount) {
-		if (isReadOnly() || epiDataParentClass == ContactDto.class && activityAsCaseCount > 0) {
+		if (isReadOnly() && activityAsCaseCount > 0) {
```

Comment:
This will make it impossible to add activities :) the second condition has to be removed because it was only relevant for contacts

Reply:
this method was deleted, replaced getPropertyDataSource(). The visibility of the field is linked to pseudonymization